gradle/libs.versions.toml

```diff
@@ -1,8 +1,8 @@
 [versions]
```

Comment:
We need also to update r8

Reply:
removed a dependency, couldn't find usages